### PR TITLE
✨(admin) allow to search users by username in django admin

### PR DESCRIPTION
**Pull request by madmatah**
## Purpose

We want to be able to search users by username in django admin. In its
current state, the User admin view is unusable with thousands of
users.

## Proposal

- [x] Add the `username` field to the search fields in the UserAdmin

### CODE REVIEW

File: src/ashley/admin.py [hunk 1]
```diff
@@ -7,3 +7,5 @@
 @admin.register(User)
 class UserAdmin(admin.ModelAdmin):
     """Admin class for Ashley's User model."""
+
+    search_fields = ["username"]
```
Comment (sampaccoud) [inline src/ashley/admin.py:11]: Why not add also email, uid, etc while you're at it

**Review by carofun — APPROVED** [review]
Great move !!!

Small question? Do we need to write django instead of Django with a capital letter in your commit message

File: src/ashley/admin.py (hunk 1 above)
Comment (madmatah) [reply src/ashley/admin.py]: You're right.
I just added the same fields than in Django's `UserAdmin` + the one that are specific to Ashley.

**Comment by madmatah** [comment]
> Do we need to write django instead of Django with a capital letter in your commit message

@carofun Yep, I fixed it

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+
+- allow to search users by username in Django admin
```
Comment (sampaccoud) [inline CHANGELOG.md:13]:
```suggestion
- allow to search users by any of its meaningful fields in Django admin
```

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**